client/src/components/Recipes/index.js

```diff
@@ -36,21 +34,12 @@ class Recipes extends Component {
         <Row>
           <Col>
             <SearchBar fetchRecipes={this.fetchRecipes} />
```

Comment:
fetchRecipes should be a function that calls "/api/recipes?recipeName=" + search term from the user input.